packages/sanity/src/core/components/documentStatusIndicator/DocumentStatusIndicator.tsx

```diff
@@ -25,23 +26,26 @@ const Root = styled(Text)`
  * - Yellow (caution) for published documents with edits
  * - Gray (default) for unpublished documents (with or without edits)
  *
- * No dot will be displayed for published documents without edits.
+ * No dot will be displayed for published documents without edits or for version documents.
  *
  * @internal
  */
-export function DocumentStatusIndicator({draft, published}: DocumentStatusProps) {
+export function DocumentStatusIndicator({draft, published, version}: DocumentStatusProps) {
   const $draft = Boolean(draft)
   const $published = Boolean(published)
+  const $version = Boolean(version)
 
   const status = useMemo(() => {
+    if ($version) return undefined
     if ($draft && !$published) return 'unpublished'
     return 'edited'
-  }, [$draft, $published])
+  }, [$draft, $published, $version])
 
   // Return null if the document is:
   // - Published without edits
   // - Neither published or without edits (this shouldn't be possible)
-  if ((!$draft && !$published) || (!$draft && $published)) {
+  // - A version
+  if ((!$draft && !$published) || (!$draft && $published) || $version) {
```

Comment:
I'm not sure about this changes, I think we don't want to check for `version` in the `DocumentStatusIndicator`, not sure if you meant to add it or if this is because of a rebase issue.
It was removed as part of other PR

Reply:
I re-added it intentionally. My thinking here is that we never want the status indicator to reflect the status of the main document when a version is checked out; we don't care about the published version or the draft in this scenario.

That is my assumption, at least. What do you think?